src/test/compile-fail/const-pattern-irrefutable.rs

```diff
@@ -9,8 +9,8 @@
 // except according to those terms.
 
 mod foo {
-    pub const b: u8 = 2; //~ NOTE constant defined here
-    pub const d: u8 = 2; //~ NOTE constant defined here
```

Comment:
We used to emit two notes for an conflicting imported constant -- one where the conflict was imported and another where the conflict was defined. For all other conflict errors, we only emit where the conflicting imported item was imported.